harvesting/crontab

```diff
@@ -1,3 +1,4 @@
-# min	hour	day	month	weekday	command
-0	*	*	*	*	cd /app && python app.py harvest all
-*/5	*	*	*	*	cd /app && python app.py resolve
\ No newline at end of file
+# minute         hour day month	weekday command
+0                5    *   *     *       cd /app && python app.py harvest citations && python app.py harvest commits && python app.py harvest metadata
```

Comment:
Can you add a comment to explain what each of these lines does, maybe point to existing docs